files/templates/alertmanager/alertmanager.yaml

```
@@ -427,7 +427,7 @@ inhibit_rules:
   equal: [cluster_id]
 
 - source_matchers:
-    - vintage_cluster_release_before_v19=true
+    - cluster_not_running_prometheus_agent=true
```

```suggestion
    - cluster_is_not_running_prometheus_agent=true
```

Sorry about that one :)